src/core/theme/utils/typography.ts

```diff
@@ -1,20 +1,19 @@
-import {
-  cssObjectsToCss,
-  FlattenSimpleInterpolation,
-} from '../../../utils/css';
-import { typographyTokens, TypographyTokens } from '../typography';
+import { FlattenSimpleInterpolation } from 'styled-components';
+import { cssObjectsToCss, cssValueToString } from '../../../utils/css';
+import { tokens, TypograhpyDesingTokens } from 'suomifi-design-tokens';
+import { SuomifiTheme } from '../';
 
 export type TypographyTokensAsCss = {
-  [key in keyof typeof typographyTokens]: string
+  [key in keyof typeof tokens.typography]: string
 };
 
 type TypographyTokensAsCssProp = {
-  [key in keyof TypographyTokens]: FlattenSimpleInterpolation
+  [key in keyof TypograhpyDesingTokens]: FlattenSimpleInterpolation
 };
 export interface TypographyUtil extends TypographyTokensAsCssProp {}
 export class TypographyUtil {
   static instance: TypographyUtil;
-  constructor(tokens: TypographyTokens) {
+  constructor(tokens: TypograhpyDesingTokens) {
```

Comment:
Just a typo, but 'Typogra**hp**yDesi**ng**Tokens' should be 'Typogra**ph**yDesi**gn**Tokens'  instead?

Might be worth checking the other places also for 'Desi**ng**' vs 'Desi**gn**'

Reply:
This issue comes from the tokens repo. Already fixed, but has to be taken into use in ui-components.